lib/client.js

```diff
@@ -128,7 +128,13 @@ Client.prototype.request = function (method, path, data) {
             }
             resolve(res);
         });
-    }).bind(this);
+    }).bind(this)
```

Comment:
If I remember correctly bluebird is ignored in the frontend build and replaced with a generic promise implementation, @connor4312 isn't that correct?